packages/server/lib/controllers/sync.controller.ts

```diff
@@ -414,7 +417,8 @@ class SyncController {
                 throw new NangoError('failed_to_get_sync_client');
             }
 
-            const actionResponse = await syncClient.triggerAction({
+            const orchestrator = new Orchestrator(new OrchestratorClient({ baseUrl: getOrchestratorUrl() }));
```

Comment:
A bit "painful" to import all that stuff that the consumer does not really care about. Having a util like `getOrchestrator()` would be clearer imo